src/main/resources/application.yml

```
@@ -0,0 +1,22 @@
+    url: jdbc:h2:mem:test
+    driver-class-name: org.h2.Driver
+    username: sa
+    password: password
+
+jwt:
+  secret:
+    key:
+  access:
+    expiration:
+  refresh:
+    expiration:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)